vacasa/connect/connect.py

```diff
@@ -855,13 +855,17 @@ def create_cancelled_reservation(self,
 
         url = f"{self.endpoint}/v1/reservations-abandoned"
         headers = self._headers()
+        last_night = subtract_days(departure, 1)
+        if last_night is None:
+            raise ValueError("Invalid departure date", departure)
```

Comment:
Would it be easier to just raise this exception in the `subtract_days` function?  Rather than passing around `None` and checking for it

Reply:
Yes, but then the SDK couldn't provide feedback to the user on which date parameter was invalid unless I also catch the exception thrown in the `subtract_days` function.